source/type/data.h

```diff
@@ -129,8 +130,8 @@ struct ContainerTrait<type::MetaVehicleJourney> {
   */
 class Data : boost::noncopyable{
 
-    static_assert(IS_TRIVIALLY_COPYABLE(const boost::posix_time::ptime), "ptime isn't is_trivially_copyable and can't be used with std::atomic");
-    mutable std::atomic<const boost::posix_time::ptime> _last_rt_data_loaded; //datetime of the last Real Time loaded data
+    static_assert(IS_TRIVIALLY_COPYABLE(navitia::ptime), "ptime isn't is_trivially_copyable and can't be used with std::atomic");
+    mutable std::atomic<navitia::ptime> _last_rt_data_loaded; //datetime of the last Real Time loaded data
```

Comment:
Why removing the constness here ? If you really need to, do you still need it to be mutable ?

Reply:
As discussed, can't keep constness (as we change value).
And mutable is here because it's the only thing that can be changed on a const Data (and it sometimes is changed).